egs/librispeech/WSASR/README.md

```diff
@@ -0,0 +1,204 @@
+# Introduction
+
+This is a weakly supervised ASR recipe for the LibriSpeech (clean 100 hours) dataset. We train a
+conformer model using [Bypass Temporal Classification](https://arxiv.org/pdf/2306.01031.pdf) (BTC)/Omni-temporal Classification (OTC) with transcripts with synthetic errors. In this README, we will describe
+the task and the BTC/OTC training process.
+
+Note that OTC is an extension of BTC and supports all BTC functions. Therefore, in the following, we only describe OTC.
+## Task
+We propose BTC/OTC to directly train an ASR system leveraging weak supervision, i.e., speech with non-verbatim transcripts. This is achieved by using a special token $\star$ to model uncertainties (i.e., substitution errors, insertion errors, and deletion errors) 
```

Comment:
By the way, does it handle deletion errors?
The BTC paper says it handles only substitution and insertion errors, but not deletion errors.

Reply:
Yeah, the new OTC method can also handle deletions. I think Dongji has submitted the paper to ArXiv and it should be available on Monday.

@DongjiGao let's wait until the paper is on arxiv, so that you can add that link in the README, along with the BTC paper.

Reply:
Sure.